auslib/web/admin/views/rules.py

```diff
@@ -200,6 +200,9 @@ def __init__(self):
 
     @requirelogin
     def _post(self, transaction, changed_by):
+        if connexion.request.get_json().get("when", None) is None:
+            return problem(400, "Bad Request", "'when' cannot be set to null when scheduling a new change "
+                                               "for a Release")
```

Comment:
I think this should be "Rule", not "Release"?

Reply:
Thank you, I had completely missed that one.